client/components/common/FileUpload.vue

```diff
@@ -17,12 +17,15 @@
         :class="[sm ? 'v-btn v-btn--small' : 'btn btn-material btn-sm upload-button']">
         {{ label }}
       </label>
-      <span
-        v-else
-        @click="downloadFile"
-        class="file-name">{{ fileName }}
-      </span>
-      <span v-if="fileKey" @click="deleteFile" class="mdi mdi-delete delete"></span>
+      <span v-else @click="downloadFile" class="file-name">{{ fileName }}</span>
```

Comment:
Can you please try and make this into a flat Vuetify button and see what it looks like.

Reply:
I made the changes and it does look better:
![Screen Shot 2019-07-31 at 13 28 10](https://user-images.githubusercontent.com/20609558/62208441-28f3a300-b397-11e9-92c3-4911357ea377.png)